packages/console/src/pages/ApiResourceDetails/index.tsx

```diff
@@ -79,6 +84,19 @@ const ApiResourceDetails = () => {
             </div>
             <div className={styles.operation}>
               <Button title="admin_console.api_resource_details.check_help_guide" />
+              <Modal
+                isOpen={isDeleteOpen}
+                className={modalStyles.content}
+                overlayClassName={modalStyles.overlay}
```

Comment:
The usage feels a little bit strange here.  Since these styles belong to Modal, why not be auto-included within the Modal component itself?

Reply:
Add an issue to track this. LOG-1907